jerry-debugger/jerry-client.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+# Copyright 2016 University of Szeged.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import socket
+import sys
+
+PORT = 5001
+HOST = "localhost"
+
+def main():
+    try:
+        client_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        client_socket.connect((HOST, PORT))
+    except socket.error, msg:
+        print >> sys.stderr, ('Failed to create the socket. Error code: %d, Error message: %s' % (msg[0], msg[1]))
```

Comment:
Most probably this won't work with python3. I would propose to use sys.stderr.write('message\n')

Anyway, I've just found something: http://stackoverflow.com/questions/2949974/how-to-exit-a-program-sys-stderr-write-or-print

It seems sys.exit('message') is a proposed way and exits with exit code 1